docs/examples/lambda-aws/index.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use MongoDB\Client;
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('The MONGODB_URI environment variable is not set');
+$client = new Client($uri);
+$planets = $client
+    ->selectDatabase('sample_guides')
+    ->selectCollection('planets')
+    ->find(
+        [],
+        [
+            'sort' => ['orderFromSun' => 1],
+            'typeMap' => ['root' => 'array'],
+        ],
+    );
+
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>MongoDB Planets</title>
+</head>
+<body>
+    <ul>
+        <?php foreach ($planets as $planet) : ?>
+            <li><?= $planet['name'] ?></li>
```

Comment:
I'll mention that with the default type map, we can continue using array access, but also use property access to indicate that we're dealing with a document:
```suggestion
            <li><?= $planet->name ?></li>
```